makebeta

```diff
@@ -49,7 +49,23 @@ echo "Success, making beta...";
 cp -f appversion.h ${version_file}
 cp -f History.txt ${history_file}
 
-# Generate the archive
+##############################################
+# Cleaning JavaScript files in directory WWW #
+##############################################
+
+# Remove all source .js files as they should now be compressed
+find www -name *.js -exec rm '{}' \;
+
+# Remove all intermediate files ending in .min
+find www -name *.min -exec rm '{}' \;
+
+# Rename files from *.js.min.gz to *.js.gz before inclusion to archive
```

Comment:
Several questions  here.... :-)

There are lots of files **containing** .min in the WWW directory, but none **ending** with ".min". The files I remove are just intermediate files created during the make process out of the original JS files before they are compressed. They are never used in this format and should never appear in this form in the installation directory. The only files that need to be found in the installation directory are the "*.js.gz" ones. We must also make sure that there are never both the "*.js" and the corresponding "*.js.gz" in the destination directory at the same time or caching will be an issue if one only is modified.

minimfy is disabled in this version as you requested, but the code is still there. One comment to remove and it works again so we can have a few people testing it (I have already found a few issues and their fix)

If someone downloads the pre-packed binary and wants to work on the js files then you are right, he will ultimately decompress a minimifed file and all variables will have meaningless names (this feature can be disabled while still removing useless space, CR and others). But do contirubutor work on pre-packed binaries? I would think they download the full source tree and install from there. In that case it is different. They can work on the original JS file which is still in the directory. When they "make install", the JS file will be minified and compresed again before being pushed to the installation directory. If they want to debug with the real variable names, they just need to run cmake as "cmake -DCMAKE_BUILD_TYPE=**Debug**" instead of the more usual "cmake -DCMAKE_BUILD_TYPE=**Release**".
When using git, they should of course never commit one of the modified files (*.min or *.js.gz) but only on the original *.js file.

html5.appcache is not a problem. You only put there files ending in .js. When  trying to get the file, the browser looks for both the compressed and uncompressed versions. My code does not change the name of the file, but only adds a .gz extension (the *.min file is a by-product, like a *.o file for gcc), so the file is still found whatever version is in the www directory, compressed or not.

The same can be done with css files.